src/components/backend-ai-registry-list.ts

```diff
@@ -116,18 +116,12 @@ class BackendAIRegistryList extends BackendAIPage {
           --label-font-size: 11px;
         }
 
-        mwc-select.full-width {
-          width: 100%;
-        }
-
-        mwc-select.full-width.fixed-position > mwc-list-item {
-          width: 330px; // default width
-        }
-
         mwc-select#select-registry-type {
           padding-right: 10px;
           --mdc-select-fill-color: transparent;
           --mdc-theme-primary: var(--general-textfield-selected-color);
+          --mdc-menu-max-width: 200px;
+          --mdc-menu-min-width: 200px;
         }
```

Comment:
And I think the width of registry type should be 100%. It seems like It's not aligned properly.
## Before
![Screen Shot 2022-07-20 at 5 05 43 PM](https://user-images.githubusercontent.com/46954439/179933094-2d7963ef-409d-4c63-a316-68d6d3bc1454.png)

## After
![Screen Shot 2022-07-20 at 5 18 23 PM](https://user-images.githubusercontent.com/46954439/179933110-0a34eff3-96b5-4c22-8a0a-4d5a7816731a.png)

How do you think @studioego and @Sujin-Kim1 ?

Reply:
I'll just apply 100% width.